### DISCUSSION TITLE
List of 3rd party product software used in MinIO

### DISCUSSION THREAD

**Discussion by khameb** [minio/minio#15626] [Q&A] [answered]
Hi,

I am collecting list of all 3rd party product software such as golang, Prometheus, go libraries … etc. Could someone share list of it and its versions of the latest MinIO release?
Thank you!

**Comment by klauspost** [comment, marked as answer]
Main [go.mod](https://github.com/minio/minio/blob/master/go.mod) contains Go dependencies.

I believe [portal-ui/package.json](https://github.com/minio/console/blob/master/portal-ui/package.json) covers the console UI.
